overnight-battery-loan/Makefile

```diff
@@ -0,0 +1,6 @@
+all: form.pdf
+
+
+form.pdf: form.tex
```

Comment:
We're potentially going to end up with a lot of `form.pdf` files here, perhaps we could name them the same as their folder instead?